tests/func/test_checkout.py

```diff
@@ -409,44 +407,45 @@ def setUp(self):
         self._prepare_repo()
 
     def test(self):
+        encoding = sys.stderr.encoding
         with self._caplog.at_level(logging.INFO, logger="dvc"), patch.object(
-            sys, "stdout"
-        ) as stdout_mock:
-            self.stdout_mock = logger.handlers[0].stream = stdout_mock
+            sys, "stderr"
+        ) as stderr_mock:
+            self.stderr_mock = logger.handlers[0].stream = stderr_mock
+
+            if sys.version_info[:1] < (3,):
+                sys.stderr.encoding = encoding
```

Comment:
Hope this is fine. Couldn't find a more elegant `Mock`-like way of doing this after briefly scanning the docs though I feel there should be. Really not nice messing with these sort of issues https://github.com/tqdm/tqdm/issues?q=encoding

Reply:
@casperdcl A comment about it would be nice :)

Reply:
Though, it seems like we can throw out this test, because it is already tested on tqdm side, right?

Reply:
brütal prüning, sure